### ISSUE TITLE
dont closes down

### ISSUE THREAD

**Issue by greatfang12** [baffler/Transparent-Twitch-Chat-Overlay#43]
hi 

i am having problem with the program, it don't closes down completely i have to go to task manager to end the program.

thank you

**Comment by Zariwey0** [comment]
Yeah, having the same problem. It is not working properly at the moment.

Using 0.7 again.

Thank you.

**Comment by baffler** [comment]
Might need more information about this. I was assuming you might not be able to close it because I added a new setting that hides the taskbar icon and enabled it by default. I reverted that change, the setting is still there but not on by default, so you can download the updated version for [v0.8](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/releases/tag/v0.8) now to fix that.

But if you mean you are closing the application normally and it's still running. I'm not having this issue so if you can explain exactly what's happening that would help a lot.

**Comment by greatfang12** [comment]
yea i meant by i was trying to close it down normally the program, it would just close it but still stay on the right of the taskbar and i still see it running on the background. but now its works good thank you for the fix.

**Comment by greatfang12** [comment]
sorry still having same issue once i put the streamlabs widget viewer count and i then my computer went to sleep because i was away from it and now the viewer count window is gone and i tried to close the program down so i could reopen the chat and the viewer count windows but the program still working on background

**Comment by baffler** [comment]
Thanks for the info, I'll see if I can recreate the issue.

**Comment by mix3d** [comment]
As another user with the problem, the issue is possibly due to settings mismatches post-upgrades. 

I was right clicking the tray icon and trying to click "exit", a window would flash, but the app would stay open. 
I was able to fix this by unchecking "General -> Confirm When Closing"

If the Confirm setting is on by default, but there is an issue during the close action that cancels the confirmation window, you're stuck with the app not-user closeable.

**Comment by baffler** [comment]
> As another user with the problem, the issue is possibly due to settings mismatches post-upgrades.
> 
> I was right clicking the tray icon and trying to click "exit", a window would flash, but the app would stay open.
> I was able to fix this by unchecking "General -> Confirm When Closing"
> 
> If the Confirm setting is on by default, but there is an issue during the close action that cancels the confirmation window, you're stuck with the app not-user closeable.

That makes sense. That should help me track down the bug 👍
